src/server/index.js

```diff
@@ -5,7 +5,41 @@ const config = require('./config.json')
 const app = express()
 const logger = require('./logger.js')
 
+const passport = require('passport')
+const expressSession = require('express-session')
+const MongoSessionStore = require('connect-mongo')(expressSession)
+const mongoSanitize = require('express-mongo-sanitize')
+
+const dbConnection = require('./dbConnection')
+const passportStrategies = require('./passport/strategies')
+const passportStrategyUtils = require('./passport/strategyUtils')
+const LocalStrategy = require('passport-local').Strategy
+
 app.use(bodyParser.json())
+app.use(mongoSanitize())
+
+app.use(expressSession({ 
+	store: new MongoSessionStore({ mongooseConnection: dbConnection} ),
+	secret: config.session.secret, 
+	resave: config.session.resave, 
+	saveUninitialized: config.session.saveUninitialized,
+	cookie: { 
+		secure: config.session.cookie.secure,
+		expires: new Date(Date.now() + config.session.cookie.expires),
```

Comment:
Declare `new Date(Date.now() + config.session.cookie.expires)` as const. Maybe `cookieExpirationDate`?
Also, it looks like this code will run upon server startup and you'll have constant value of current time + cookie expiration date passed here. Doesn't it mean that after some time newly created cookies will be instantly expired?

Reply:
ok